api/coverage.proto

```diff
@@ -9,6 +9,9 @@ option java_outer_classname = "ApiCoverageProto";
 message OptionsData {
     string id = 1;
     repeated OptionsChart optionsChart = 2;
+    string company_id = 3;
+    string billinggroup_id = 4;
```

Comment:
@Woody1193 Also, please use camelCase for fields, not snake_case. For consistency.

Reply:
Sure. I'll fix this as well